subprojects/errorprone/src/test/java/org/mockito/errorprone/bugpatterns/MockitoMockedStaticTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.errorprone.bugpatterns;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class MockitoMockedStaticTest {
+
+    private CompilationTestHelper compilationTestHelper;
+
+    @Before
+    public void setup() {
+        compilationTestHelper =
+            CompilationTestHelper.newInstance(MockitoMockedStatic.class, getClass());
+    }
+
+    @Test
+    public void mockedStaticWithMockField_shouldError() {
+        compilationTestHelper
+            .addSourceLines(
+                "input.java",
+                "import org.mockito.Mock;",
+                "import org.mockito.MockedStatic;",
```

Comment:
Think you need to import org.mocktio.Mockito here to be able to call `Mockito.mockStatic`

Reply:
ah yes I do seem to have forgotten that, I will add it. Thanks!

Reply:
Added!